lib/helpers/updateChangelog.js

```diff
@@ -22,7 +22,7 @@ const writeToFile = (commits = [], version, exists) => {
   commits.forEach((commithash) => {
     const commitInfo = gitCommitInfo({ commit: commithash, cwd });
 
-    stream.write(`* ${commitInfo.shortHash} ${commitInfo.message} (${commitInfo.author})\n`);
+    stream.write(`* ${commitInfo.shortHash} ${commitInfo.message.split('\n')[0]} (${commitInfo.author})\n`);
```

Comment:
The only thing what would not work is, if the commit message is longer than 72 characters. GitHub will automatically make a break. But it is hard to detect except if its a squash in a PR.